service/trust/delegating_key_service.go

```diff
@@ -188,6 +189,9 @@ func (d *DelegatingKeyService) getKeyManager(name string) (KeyManager, error) {
 		return nil, fmt.Errorf("configured default key manager '%s' not found (no factory/cache entry)", name)
 	}
 
-	d.l.Debug("Key manager not found by name, falling back to default", "requestedName", name, "configuredDefaultName", currentDefaultMode)
+	d.l.Debug("key manager not found by name, falling back to default",
+		slog.String("requested_name", name),
```

Comment:
Should we be more consistent with camelCase v. snake_case?

Reply:
Consistency is the goal, as casing of keys is a sloglint setting: https://github.com/opentdf/platform/pull/2462/files/d97117a09ae1f90144b8f74bd65864a99be8d3c9#diff-9917ddc9f1c3304218f7269265b746d997c5c0615478177b5fceecd33ef47cb5R136